practice_javascript.js

```diff
@@ -1,3 +1,5 @@
 console.log("hello world!")
 
-//I wrote a function that does what I said earlier
\ No newline at end of file
+//I wrote a function that does what I said earlier
+
+//add new feature below this comment
```

Comment:
This feature looks ok, but I think I prefer it this way
```suggestion
//add new feature below this comment, with documentation
```